web/ascii_art_web/templates/img-upload.html

```diff
@@ -5,10 +5,26 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>image to ascii converter</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
     <link rel="stylesheet" href="{{ url_for('static', filename='css/style.css') }}">
 </head>
 
 <body>
+    <main role="main" class="container">
```

Comment:
The main tag messes up with existing page's styling. Not sure if it's just a me-thing?
How does it appear on your side?

Reply:
you should be able to override the bootstrap css with the static style sheet that's why I put the bootstrap above the custom style sheet you have